kotlin/samples/hello-workflow-fragment/src/main/java/com/squareup/sample/helloworkflowfragment/HelloWorkflowFragment.kt

```diff
@@ -21,10 +21,10 @@ import com.squareup.workflow.ui.ViewRegistry
 import com.squareup.workflow.ui.WorkflowFragment
 import com.squareup.workflow.ui.WorkflowRunner
 
-class HelloWorkflowFragment : WorkflowFragment<Unit, Unit>() {
+class HelloWorkflowFragment : WorkflowFragment<Unit, Nothing>() {
```

Comment:
Nice catch

Reply:
Thank the compiler. A bonus of our new invariance.